src/boot.S

```diff
@@ -229,8 +229,20 @@ b       exc_handler
 .endm
 
 synchronous_el0:
```

Comment:
one thought I had was that we could just handle exceptions in code, minimizing the amt of assembly stuff we need to write. lmk ur thoughts.

check out: https://github.com/BajuMcBites/OStrich/commit/2a7b19aedd2c5c046dbc838217bc9294e7036383